wordpress-mobile/WordPress-iOS
# Reader: Add UI for the Reader tags feed

Part of #23013

## Description

Adds the UI for the "Tags" feed.

Future PRs:
- Card actions
- "More" end cell
- Accessibility
- Empty feed state
- Fetching posts for tags
- Loading placeholders
- Consolidating display functions between `ReaderPostCardCell` & `ReaderTagCell`
- Analytics

## Video

<details><summary>iPhone</summary>

https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/b930721a-2731-444e-aab0-c9279180383e
</details>

<details><summary>iPad</summary>

https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/3c3ecade-3f95-48fc-ab49-3bfd5de17280
</details>


## Testing

**Test setup:**
You'll need some cached tag posts to see any UI:
- Navigate to `Discover`
- Tap on a post
- Tap on one of the tags for that post
- If you aren't following it, follow the tag

**To test:**
- Launch Jetpack and login
- Complete the **Test setup**
- Navigate to the "Your Tags" Reader feed
- Scroll to the tag which has posts saved locally
- 🔎 **Verify** the UI matches the designs for the new cell
- Find a post with no featured image
- 🔎 **Verify** the text is expanded
- Find a post where the content doesn't fit the full height
- 🔎 **Verify**:
  - The number of likes/comments label is aligned to the bottom
  - The like button and menu button are aligned to the bottom

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:


Testing checklist:
